usagov/usagov-2021
# USAGOV-2027 Add prod config file for GSA Auth

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-2027

## Description


Added gsaauth config file for prod environment

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions

- Can only be tested properly in prod (See Validation Steps)
- On all other CF envs:  no changes should happen, and (1) the User Login process should continue to be handled by GSAAuth, and (b) User Login process should be tested to ensure it still works.

### Change Requirements



### Validation Steps

- On prod:  After deployment, the User Login process should now be handled by GSAAuth instead of SecureAuth

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
